src/cli/peakdetector/peakdetectorcli.cpp

```diff
@@ -703,6 +705,50 @@ void PeakDetectorCLI::loadSamples(vector<string>& filenames)
 #endif
 }
 
+void PeakDetectorCLI::alignSamples(const int& method)
+{
+    if (mavenParameters->samples.size() > 1) {
+        switch (method) {
+        case 1: {
+            cerr << "Starting OBI-WARP alignment" << std::endl;
+            // TODO: move the hard coded values in  default_settings.xml and
+            // instead of using obi params make use mavenParameters to access all
+            // the values.
+            ObiParams params(
+                "cor", false, 2.0, 1.0, 0.20, 3.40, 0.0, 20.0, false, 0.60);
+            Aligner mzAligner;
+            mzAligner.alignWithObiWarp(
+                mavenParameters->samples, &params, mavenParameters);
+        }
+        break;
+
+        case 2: {
+            mavenParameters->writeCSVFlag = false;
+            peakDetector->processFeatures();
```

Comment:
PolyFit had the option to align groups wrt compound search or mass slices. Has that option been removed now?

Reply:
No, in the GUI, you can still chose the method for detecting your reference groups. This is CLI. We never offered that kind of control here.